### ISSUE TITLE
Add continuous integration

### ISSUE THREAD

**Issue by Ziriax** [WonderMediaProductions/Maya2glTF#110]
Maybe use Travis CI?

At least we should:
- do a build on Windows, Mac, Linux
- create a setup ZIP file
- export a set of test scenes
- compare the exports with an approved set

**Comment by fire** [comment]
In theory, we should attempt to publish a build every possible build and on tags.

**Comment by fire** [comment]
Dev kit locations:


Maya 2019 Update 2 

* https://autodesk-adn-transfer.s3-us-west-2.amazonaws.com/ADN+Extranet/M%26E/Maya/devkit+2019/Autodesk_Maya_2019_2_Update_DEVKIT_Windows.zip
* https://autodesk-adn-transfer.s3-us-west-2.amazonaws.com/ADN+Extranet/M%26E/Maya/devkit+2019/Autodesk_Maya_2019_2_Update_DEVKIT_Mac.dmg
* https://autodesk-adn-transfer.s3-us-west-2.amazonaws.com/ADN+Extranet/M%26E/Maya/devkit+2019/Autodesk_Maya_2019_2_Update_DEVKIT_Linux.tgz

Maya 2018 Update 6

* https://s3-us-west-2.amazonaws.com/autodesk-adn-transfer/ADN+Extranet/M%26E/Maya/devkit+2018/Autodesk_Maya_2018_6_Update_DEVKIT_Windows.zip
* https://s3-us-west-2.amazonaws.com/autodesk-adn-transfer/ADN+Extranet/M%26E/Maya/devkit+2018/Autodesk_Maya_2018_6_Update_DEVKIT_Mac.dmg
* https://s3-us-west-2.amazonaws.com/autodesk-adn-transfer/ADN+Extranet/M%26E/Maya/devkit+2018/Autodesk_Maya_2018_6_Update_DEVKIT_Linux.tgz

**Comment by fire** [comment]
@Ziriax Can we make a release soon? I have an incoming PR, but it'll be less confusing to others when master is synced with develop.

**Comment by Ziriax** [comment]
Yes I would like to release 1.0 soon, but I'm still busy with other work... But you are right, develop should become master asap!

But I'm only able to really test on Windows myself...

Linux support is not yet finished, you did give some comments, but I haven't done any work on that, as I'm really a Windows guy...

One thing I noted in your latest changes (thanks again for all your work BTW), is that just building with Visual Studio doesn't pick up the current GIT versions, so when making a release, one should use CMake for building.

If you have spare time, feel free to provide a PR to release develop, if one is needed at all :)
